seatunnel-dist/release-docs/LICENSE

```diff
@@ -590,161 +524,162 @@ The text of each license is the standard Apache 2.0 license.
+     (The Apache Software License, Version 2.0) reindex (org.elasticsearch.plugin:reindex-client:6.3.1 - https://github.com/elastic/elasticsearch)
      (Apache-2.0) resilience4j (io.github.resilience4j:resilience4j-bulkhead:1.3.1 - https://github.com/resilience4j/resilience4j)
-     (Apache-2.0) resilience4j (io.github.resilience4j:resilience4j-core:1.3.1 - https://github.com/resilience4j/resilience4j)
+     (The Apache Software License, Version 2.0) rest (org.elasticsearch.client:elasticsearch-rest-client:6.3.1 - https://github.com/elastic/elasticsearch)
+     (The Apache Software License, Version 2.0) rest-high-level (org.elasticsearch.client:elasticsearch-rest-high-level-client:6.3.1 - https://github.com/elastic/elasticsearch)
      (The Apache Software License, Version 2.0) Retrofit (com.squareup.retrofit2:retrofit:2.9.0 - https://github.com/square/retrofit)
-     (Apache-2.0) scala-xml (org.scala-lang.modules:scala-xml_2.11:1.3.0 - http://www.scala-lang.org/)
+     (Apache 2) RoaringBitmap (org.roaringbitmap:RoaringBitmap:0.5.11 - https://github.com/lemire/RoaringBitmap)
      (Apache 2) scalaj-http (org.scalaj:scalaj-http_2.11:2.3.0 - http://github.com/scalaj/scalaj-http)
-     (Apache License Version 2.0) SIGAR (org.hyperic:sigar:1.6.5.132 - http://sigar.hyperic.com/)
-     (Apache License, Version 2.0) com.yahoo.datasketches:sketches-core (com.yahoo.datasketches:sketches-core:0.9.0 - https://datasketches.github.io/sketches-core/)
+     (The Apache License, Version 2.0) seatunnel-api-base (org.apache.seatunnel:seatunnel-api-base:2.0.5-SNAPSHOT - https://github.com/apache/incubator-seatunnel/seatunnel-apis/seatunnel-api-base)
+     (The Apache License, Version 2.0) seatunnel-api-flink (org.apache.seatunnel:seatunnel-api-flink:2.0.5-SNAPSHOT - https://github.com/apache/incubator-seatunnel/seatunnel-apis/seatunnel-api-flink)
+     (The Apache License, Version 2.0) seatunnel-api-spark (org.apache.seatunnel:seatunnel-api-spark:2.0.5-SNAPSHOT - https://github.com/apache/incubator-seatunnel/seatunnel-apis/seatunnel-api-spark)
+     (The Apache License, Version 2.0) seatunnel-common (org.apache.seatunnel:seatunnel-common:2.0.5-SNAPSHOT - https://github.com/apache/incubator-seatunnel/seatunnel-common)
+     (The Apache License, Version 2.0) seatunnel-config (org.apache.seatunnel:seatunnel-config:2.0.5-SNAPSHOT - https://github.com/apache/incubator-seatunnel/seatunnel-config)
+     (The Apache License, Version 2.0) seatunnel-connector-flink-console (org.apache.seatunnel:seatunnel-connector-flink-console:2.0.5-SNAPSHOT - https://github.com/apache/incubator-seatunnel/seatunnel-connectors/seatunnel-connector-flink-console)
+     (The Apache License, Version 2.0) seatunnel-connector-flink-doris (org.apache.seatunnel:seatunnel-connector-flink-doris:2.0.5-SNAPSHOT - https://github.com/apache/incubator-seatunnel/seatunnel-connectors/seatunnel-connector-flink-doris)
```

Comment:
these unwanted

Reply:
Hi @CalvinKirs, thanks for your comment. These are removed.